### ISSUE TITLE
Plugins: Font sizes and weights in the sidebar

### ISSUE THREAD

**Issue by jasmussen** [WordPress/wordpress.org#297]
In the plugin sidebar, there appears to be a mix of font sizes and font weights:

![Image](https://github.com/WordPress/wordpress.org/assets/1204802/e531436d-0926-42ea-bba7-1c5f7fb8b239)

Note specifically the "Advanced View" (which could be "Advanced view") the font weight looks larger. 

This sidebar from [the forums design](https://www.figma.com/file/3eK690rwPubLvTEvJIJKRO/Forums?type=design&node-id=902%3A1277&mode=design&t=hRsILsrTP1mvtsza-1) isn't a 1:1 design, but the principles are the same as far as font sizes, weights, styles, etc:

![Image](https://github.com/WordPress/wordpress.org/assets/1204802/f012d16e-75e9-409b-8cef-7521315bf0ed)

Bonus: Replace the external link icon with ↗

**Comment by StevenDufresne** [comment]
What about taking the cue from the [showcase component](https://wordpress.org/showcase/tonal/)? 

(Also noting [this conversation](https://github.com/WordPress/wordpress.org/issues/262#issuecomment-2060040400) about A11y with the current showcase sidebar. I think we could underline the links without causing much visual disruption.)

<img width="337" alt="Screenshot 2024-05-01 at 3 54 56 PM" src="https://github.com/WordPress/wordpress.org/assets/1657336/c3a60dbb-2690-490a-8cd0-b4607f2eb60b">

**Comment by jasmussen** [comment]
Yes, I think it's okay to underline the links inside the sidebar module. I think there's more nuance in some of the showcase specific feature boxes that we could potentially make buttons instead of links inside boxes, but specifically for this issue and the example above, they can be the same I'd agree.
